batch/batch/server/database.py

```diff
@@ -100,7 +100,7 @@ def make_where_statement(items):
     values = []
     for k, v in items.items():
         if isinstance(v, list):
-            if len(v) == 0:
+            if not v:
```

Comment:
I thought that we had settled on explicit value checking. It feels more correct from that perspective to avoid explicit boolean conversion. Please correct me if I'm wrong.

Reply:
I changed pylint settings to ignore this instead.